package.json

```diff
@@ -34,11 +55,18 @@
     "safari"
   ],
   "contributors": [
-    "rishabhpoddar"
+    "rishabhpoddar",
+    "NkxxkN"
   ],
   "license": "Apache-2.0",
   "bugs": {
     "url": "https://github.com/supertokens/supertokens-auth-react/issues"
   },
-  "homepage": "https://github.com/supertokens/supertokens-auth-react#readme"
+  "homepage": "https://github.com/supertokens/supertokens-auth-react#readme",
+  "dependencies": {
+    "@emotion/core": "^10.0.35",
+    "emotion": "^10.0.27",
+    "emotion-server": "^10.0.27",
```

Comment:
emotion server is a peer dependency of react-shadow if we want to use emotion with react-shadow.